### ISSUE TITLE
EFI 1.1 compliance

### ISSUE THREAD

**Issue by BurnedPinguin** [maharmstone/quibble#95]
Is it in the plans to make this bootloader or at least a separate one-off build of it that works on EFI 1.1 firmware? I would like to use it with some of Apple's early Macs and I think it would be very significant if this bootloader can do this one day, because nothing else can, and this is the only active project I found that I could ask whether they would intend to do this.

**Comment by maharmstone** [comment]
Probably not. It's not the sort of thing you can really do without having access to the hardware in question. Sorry.

That said, have you given it a go? I'm fairly sure GOP won't work, but I don't know about everything else.

**Comment by BurnedPinguin** [comment]
Yeah I tried quibble x86 and x64 binaries, they just lock up the computer when I select the device that is made bootable with them (it locks up at the option key disk select screen, not even a black screen!)

The regular EFI version of windows boot manager doesn't work either. A solution here would be something that could "finish" Apple's EFI implementation - a stopgap EFI executable that would fill in for what its missing and that the EFI version of windows boot manager needs.

**Comment by kocoman1** [comment]
can this run on corebooted tianocore mbp10,1 or mbp8,2? thx

**Comment by maharmstone** [comment]
@kocoman1 - I don't know. Try it and let me know.

**Comment by mirh** [comment]
https://github.com/torvalds/linux/commit/abb2bafd295fe962bbadc329dbfb2146457283ac, https://github.com/rhboot/shim/pull/364 and https://github.com/acidanthera/bugtracker/issues/1280 show a lot of quirks.
(even [without](https://mjg59.livejournal.com/132477.html) [all](https://mjg59.dreamwidth.org/3244.html) [the](https://mjg59.dreamwidth.org/4957.html) [rest](http://www.rodsbooks.com/ubuntu-efi/index.html) which may or may not have been just part of the original linux efi spinup)
The probably biggest problem is that I [understand](https://mjg59.dreamwidth.org/12037.html) the bootloader needs to be able to work with HFS? 

https://superuser.com/questions/671660/graphics-card-not-working-on-windows-8-1-on-mac
Anyway, this should already be possible to test/try with rEFInd. The only legit inconvenience is that the gpu has to be more or less manually set up.
